ifu_session/Cube_Analysis_Webbinar_solutions.ipynb

```diff
@@ -8,14 +8,14 @@
     "\n",
```

Comment:
<p>Not sure if the commented out cells are needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>